dongkyu/initial-settings-express/myapp/app.js

```diff
@@ -82,17 +80,18 @@ app.get("/posts", async (req, res) => {
             posts.content
       FROM users
       INNER JOIN posts
-      ON users.id = posts.user_id`
-,(err, rows) => {
-  res.status(200).json({data : rows});
-  })
-})
+      ON users.id = posts.user_id
+    `
+  );
+
+  res.status(200).json({ data: rows });
+});
+
+app.get("/postings/:userId", async (req, res) => {
+  const { userId } = req.params;
 
-  app.get("/postings/:userId", async (req, res) => {
-    const { userId } = req.params;
-  
-    await appDataSource.query(
-      `SELECT
+  await appDataSource.query(
```

Comment:
111번에서 `res.status(200).json({ data: rows });`로 작성하셨기에 다음과 같이 수정 되어야 합니다!
```suggestion
  const rows = await appDataSource.query(
```